uber/tasks/email.py

```diff
@@ -200,15 +200,15 @@ def send_automated_emails():
                                 quantity_sent += 1
                             else:
                                 unapproved_count += 1
-                            if datetime.now() - automated_email.last_send_time > (expiration / 2):
-                                automated_email.last_send_time = datetime.now()
-                                session.add(automated_email)
-                                session.commit()
-
-                    automated_email.unapproved_count = unapproved_count
-                    automated_email.currently_sending = False
-                    session.add(automated_email)
-                    session.commit()
+                    if datetime.now() - automated_email.last_send_time > (expiration / 2):
```

Comment:
Re-reading this, I think this unindent will actually slightly increase the load on the server, as it will check for last send time on every model instance instead of just the ones that newly qualify for the email